### DISCUSSION TITLE
Using biomod2 with ordinal abundance data

### DISCUSSION THREAD

**Discussion by trehill** [biomodhub/biomod2#521] [Q&A]
I'm working on building Species Distribution Models (SDMs) for a project using an ordinal abundance (ie. species occurence/response variable is ordinal and binned into categories such as 'None', 'Few', and 'Many'). I would like to build SDM predictions to reflect this ordinal abundance rather than being simplified into presence/absence or presence-only data. Is this possible using biomod2?

**Comment by HeleneBlt** [comment]
Hi !

I'm delighted to announce that we've recently added abundance in `biomod2` ! 🎉 
I wouldn't say it's perfect yet (especially the ordinal data, which is a bit tricky!). but all the information are [here](https://biomodhub.github.io/biomod2/articles/vignette_Abundance.html).
Any feedback will be greatly appreciated ! 
Don't hesitate if you have any question.

Hélène
